nav-app/src/app/help/help.component.html

```diff
@@ -462,8 +470,39 @@ <h3 id="help_IDs_Names">
             <p>
                 In the side and flat layouts, you can change what is shown inside of the technique cells. Enabling "show names" (enabled by default) will show technique and tactic names on each cell of the matrix. Enabling "show IDs" (disabled by default) will show ATT&CK IDs (e.g "T1000" for techniques, or "TA1000" for tactics) on each cell of the matrix.
 
-                These controls can be toggled independantly and turned off entirely to remove cell labels entirely. The mini layout forces both of these controls to be disabled.
+                These controls can be toggled independently and turned off entirely to remove cell labels entirely. The mini layout forces both of these controls to be disabled.
+            </p>
+            <h2 id="help_Aggregate_Scores">
+                Aggregate Scores
+            </h2>
+            <p>
+              Aggregate scores will combine the scores of a technique and all of its sub-techniques, and can be calculated using either the average, max, min or sum function.
+              They will display only on techniques with sub-techniques. The display of aggregate scores in the matrix view and the score calculations can be customized via the "Matrix Configuration" dropdown menu.
+            </p>
+            <h3 id="help_Show_Hide_Aggregate">
+              Showing or Hiding Aggregate Scores
+            </h3>
+            <p>
+              Aggregate scores in the view are displayed in the tooltip when hovering over a technique, and are hidden by default. When this is enabled, the technique's background color will be calculated using the aggregate score by default.
             </p>
+            <h3 id="help_Count_Unscored">
+              Counting Unscored Techniques
+            </h3>
+            <p>
+              Techniques are unscored by default (they have a score of 0, which is not displayed in the technique's tooltip). The ability to include the unscored techniques in the calculation of the aggregate score can be toggled via the "Matrix Configuration" dropdown.
+            </p>
+            <h3 id="help_Choose_Aggregate">
+              Choosing an Aggregate Function
+            </h3>
+            <p>
+              There are 4 available functions to calculate the aggregate score: average, min, max, and sum.
+            </p>
+            <ul>
+              <li>Average: if including unscored techniques, will add up the scores of a technique and all of its sub-techniques, and divide by all; if not including unscored techniques, will add up the only the nonzero scores of a technique and its sub-techniques, and divide by the count of non-zero scores.</li>
+              <li>Min: if including unscored techniques, will take the minimum of the scores of a technique and all of its sub-techniques; if not including unscored techniques, will take the minimum of only the nonzero scores of a technique and its sub-techniques.</li>
```

Comment:
I would recommend taking the two "ifs" based on the "count unscored techniques" button and making them sub-bullets for readability.